pkg/packagekit/updates.jsx

```diff
@@ -1075,7 +1075,15 @@ class OsUpdates extends React.Component {
                     this.setState(nextState);
                 })
                 .catch((exception, data) => {
-                    console.error(`Tracer failed: "${JSON.stringify(exception)}", data: "${JSON.stringify(data)}"`);
+                    // common cases: this platform does not have tracer installed
+                    if (!exception.message?.includes("ModuleNotFoundError") &&
+                        // or supported (like on Arch)
+                        !exception.message?.includes("UnsupportedDistribution") &&
+                        // or polkit does not allow it
+                        exception.problem !== "access-denied" &&
+                        // or the session goes away while checking
+                        exception.problem !== "terminated")
+                        console.error(`Tracer failed: "${JSON.stringify(exception)}", data: "${JSON.stringify(data)}"`);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19503-20231018-101508-5f9f5ce3-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)